com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -9,11 +9,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Added support for user-selected upscaling filters. Current options are automatic, bilinear, and nearest-neighbor.
 - Added batch mode support for the converters.
+- Soft shadow quality. Low, PCF 3x3 recommended for mobile. Medium, Tent 5x5. High, Tent 7x7.
 
 ### Changed
 - Re-added the menu button to be able to convert selected materials.
 - Reverted intermediate texture behavior.
 - Shader Variant Log Level moved from the URP Asset to URP Global Settings.
+- Remove SHADER_API_MOBILE from shaders in cases where it affects quality.
+- Remove SHADER_HINT_NICE_QUALITY from shaders.
+- Remove low quality light fade.
+- Remove implicit SHADER_QUALITY_LOW. It now must be defined in .shader file to opt-in. SHADER_QUALITY_LOW affects particle, grass and nature view vector normalization, LOD crossfade, SH eval vertex/pixel.
```

Comment:
We should document this properly, otherwise it's better to remove until we expose shader quality tiers as without docs majority of users won't benefit from this.

Reply:
Removed.